code-security/admin_guide/get-started/connect-your-repositories/add-private-registries.adoc

```diff
@@ -0,0 +1,79 @@
+:topic_type: task
+
+[.task]
+== Add Private Registries to Prisma Cloud Code Security
+
+Private Registries integration with Prisma Cloud console enables you to find an accurate dependency tree of the packages in your private registries. The accuracy into your registries, ensures you have better fix suggestions for potentially vulnerable CVE.
+On Prisma Cloud scans are run on  both Private registries and public registries giving you a more concise scan results and fix suggestions.
+To remediate vulnerabilities found on Private registries see results and fix suggestions on *Projects*. Alternatively, you can make an informed decision by seeing the scan results on *Supply Chain*, where the graph displays the dependency tree of the latest private integration.
+
+[.procedure]
+
+. Verify prerequisites.
++
+For Private Registries integration on Prisma Cloud Code Security, you need permissions and must have Prisma Cloud IP addresses and hostname on an allow list.
++
+* Access to the Prisma Cloud console.
++
+Ensure  your private registries have access to the Prisma Cloud IP addresses and hostname. You need access to 44.237.183.44 and 35.82.113.182. For more information see https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/get-started-with-prisma-cloud/enable-access-prisma-cloud-console.html#id7cb1c15c-a2fa-4072-%20b074-063158eeec08[enable access to the Prisma Cloud Console].
++
+* Permissions required to see scan results after integration.
```

Comment:
We need to add that the user must have credentials for Artifactory with either read only (BC) or admin (Cider) permissions.